dev/NavigationView/NavigationView_rs1_themeresources.xaml

```diff
@@ -80,6 +80,7 @@
             <StaticResource x:Key="NavigationViewButtonForegroundPointerOver" ResourceKey="TextFillColorPrimaryBrush" />
             <StaticResource x:Key="NavigationViewButtonForegroundPressed" ResourceKey="TextFillColorSecondaryBrush" />
             <StaticResource x:Key="NavigationViewButtonForegroundDisabled" ResourceKey="TextFillColorDisabledBrush" />
+            <StaticResource x:Key="NavigationViewItemHeaderForeground" ResourceKey="TextFillColorPrimaryBrush"/>
```

Comment:
nit: move this property somewhere around NavigationViewItemSeparatorForeground in terms of ordering for better readability